QuackNBOOM/QuackNBOOM/Bomb.swift

```diff
@@ -0,0 +1,109 @@
+//
+//  Bomb.swift
+//  QuackNBOOM
+//
+//  Created by Ngo Tuyetnhi N. and Benoit Neriah R. on 3/4/18.
+//  Copyright © 2018 Neriah and Neenee. All rights reserved.
+//
+// Bomb is derived from GameObject. It's weapon used by the player to kill the ducks.
+//      The bomb goes to the tapped area and explodes after a countdown.
+//      The explosion works by hiding the bomb is displaying the explosion - two different sprites
+
+import Foundation
+import SpriteKit
+
+class Bomb: GameObject {
+    let explosionSprite: SKSpriteNode           // Another sprite for the explosion
+    let startExplodeCountdown = CGFloat(0.5)    // The value as to what to reset the countdown to
+    var explodeCountdown = CGFloat(0.5)         // The explosionCountdown explodes the bomb when it reaches 0. When the bomb has been exploded, it's set to -100
+    
+    
+    
+    // Initializes both sprites and the attributes
+    //
+    // - Parameters:
+    //      - imagePath: bomb.sprite image
+    //      - explosionImagePath: explosionSprite image
+    required init(imagePath: String, explosionImagePath: String){
+        // Set images
+        explosionSprite = SKSpriteNode(imageNamed: explosionImagePath)
+        super.init(imagePath: imagePath)
+        
+        // Set attributes
+        zPos = 2.0
+        scale = CGSize(width: 0.75, height: 0.75)
+        
+        // Set the sprites' visibility
+        sprite.isHidden = false
+        explosionSprite.isHidden = true
+    }
+    
+    // Initializer. Automatically sets the explosion sprite
+    //
+    // - Parameter: imagePath: bomb.sprite image
+    required init(imagePath: String){
```

Comment:
Like so:
```
convenience init(imagePath: String) {
     self.init(imagePath: imagePath, explosionImagePath: "Explode")
}
```

Reply:
**This requested change has been successfully implemented.**
**Code Change:**
```swift
convenience required init(imagePath: String){
     self.init(imagePath: imagePath, explosionImagePath: "Explode")
}
```